### PR TITLE
Chore/replace rison with json

### PR DESCRIPTION
**Pull request by nitrosx**
## Description

Remove rison dependency

## Motivation

We wanted to removed the dependency from rison library. All the interested files have been changed to use JSON instead of rison. 

## Fixes:

* angular.json
* package-lock.json
* package.json
* src/app/datasets/anonymous-dashboard/anonymous-dashboard.component.ts
* src/app/datasets/dashboard/dashboard.component.ts
* src/app/jobs/jobs-dashboard/jobs-dashboard.component.ts
* src/app/logbooks/logbooks-dashboard/logbooks-dashboard.component.spec.ts
* src/app/logbooks/logbooks-dashboard/logbooks-dashboard.component.ts
* src/app/policies/policies-dashboard/policies-dashboard.component.spec.ts
* src/app/policies/policies-dashboard/policies-dashboard.component.ts
* src/app/proposals/proposal-dashboard/proposal-dashboard.component.ts
* src/app/publisheddata/publisheddata-dashboard/publisheddata-dashboard.component.ts
* src/app/samples/sample-dashboard/sample-dashboard.component.ts
* src/app/shared/modules/breadcrumb/breadcrumb.component.ts
* src/app/state-management/effects/logbooks.effects.ts

## Changes:

* removed dependency in package.json and angular.js
* migrated from rison encode/decode to JSON stringify/parse

## Tests included/Docs Updated?

- [x] Included for each change/fix?
- [x] Passing? (Merge will not be approved unless this is checked) 
- [ ] Docs updated?
- [ ] New packages used/requires npm install? 
- [ ] Toggle added for new features?

### CODE REVIEW

File: src/app/datasets/anonymous-dashboard/anonymous-dashboard.component.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ import {
 import { combineLatest, Subscription } from "rxjs";
 
 import * as deepEqual from "deep-equal";
-import * as rison from "rison";
+//import * as rison from "rison";
```
Comment (henrikjohansson712) [inline src/app/datasets/anonymous-dashboard/anonymous-dashboard.component.ts:20]: This commented out import should be removed.

File: src/app/datasets/anonymous-dashboard/anonymous-dashboard.component.ts [hunk 2]
```diff
@@ -111,7 +112,8 @@ export class AnonymousDashboardComponent implements OnInit, OnDestroy {
         .pipe(
           map((params) => params.args as string),
           take(1),
-          map((args) => (args ? rison.decode<DatasetFilters>(args) : {}))
+          //map((args) => (args ? rison.decode<DatasetFilters>(args) : {}))
```
Comment (henrikjohansson712) [inline src/app/datasets/anonymous-dashboard/anonymous-dashboard.component.ts:115]: This commented out line should be removed.

**Review by henrikjohansson712 — CHANGES_REQUESTED** [review]
Looks good! Just found two commented out lines that should be removed.

File: src/app/datasets/anonymous-dashboard/anonymous-dashboard.component.ts (hunk 2 above)
Comment (nitrosx) [reply src/app/datasets/anonymous-dashboard/anonymous-dashboard.component.ts]: Done.

**Review by henrikjohansson712 — APPROVED** [review]
Looks great!

**Pull request merged**